images/installer/Dockerfile.ci

```diff
@@ -1,15 +1,14 @@
 # This Dockerfile is a used by CI to publish openshift/origin-v4.0:installer
 # It builds an image containing only the openshift-install and terraform binaries.
```

Comment:
This comment is no longer accurate as the image does not contain terraform binaries.

Reply:
done with https://github.com/openshift/installer/commit/e033d44fcb6b374fdf8ca6492dc4e73d0cea97ce -> https://github.com/openshift/installer/commit/97482d5c8e03874dda28a4273596bc7bfd7a2c45